assets/js/index.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+
+  var hamburger = {
```

Comment:
Is this valid JavaScript for all the browsers?

Reply:
no idea. You tell me :trollface:

Reply:
What are the supported browsers for this site?

Reply:
Any way I think is this and I think it's ok https://caniuse.com/#search=variable%20block forget it :)